test/verify/files/openshift.kubeconfig

```diff
@@ -1,7 +1,8 @@
 apiVersion: v1
 clusters:
 - cluster:
-    certificate-authority-data: LS0tLS1CRUdJTiBDRVJUSUZJQ0FURS0tLS0tCk1JSUM2akNDQWRLZ0F3SUJBZ0lCQVRBTkJna3Foa2lHOXcwQkFRc0ZBREFtTVNRd0lnWURWUVFEREJ0dmNHVnUKYzJocFpuUXRjMmxuYm1WeVFERTBPVEEyT0RFMk16RXdIaGNOTVRjd016STRNRFl4TXpVd1doY05Nakl3TXpJMwpNRFl4TXpVeFdqQW1NU1F3SWdZRFZRUUREQnR2Y0dWdWMyaHBablF0YzJsbmJtVnlRREUwT1RBMk9ERTJNekV3CmdnRWlNQTBHQ1NxR1NJYjNEUUVCQVFVQUE0SUJEd0F3Z2dFS0FvSUJBUURUYzNlQkRpQnZwUzFWWFo2VEhkaFAKYWJ4RENuSFdsb2VTZDMveERqaVBlRmpVN1RKeTJaUzNGT3J1cUVBTytKdGJWd0RhcU5VNkNEV1hXQ0FtK0dtdApBcE5YcG5hUlBVNDk2Vmx1b2RZZUV2a0FmWXdzZjM1a0s5TWxYbDUvY1AwYUE1UDNCbWRpT2kzdlBwcmxJeUtCClpIbzhSVU1YaXJyaHI0QjYzY2NKNHJMaDFQR28yR0tudFJyLzNnc0ZuRGlEQzFsTEZERDRIUU5GRnlTWFZRL3EKVHlxY1NZR3V2OGprRnQzem9nOENhckpHSG1PTzVYL25HY214OTJObUpTNHUwREJnRXNRaFhINXZadVBGQ3p6cQpVZktWc1M1RjJJL3AzdVRoK0RSUXYyMFZ1VTNCUUJhREplZkpUUUJ4eW9uYW5hRmg5S2YwV3gxb3dDdWtCM1dMCkFnTUJBQUdqSXpBaE1BNEdBMVVkRHdFQi93UUVBd0lDcERBUEJnTlZIUk1CQWY4RUJUQURBUUgvTUEwR0NTcUcKU0liM0RRRUJDd1VBQTRJQkFRQnJ5elFaemFkc3FFL0JEY1VlVUE2d21QUHk0MGMyOXVDazlPRzdPL0M4WGFpNwpSdWxDMXlzcmZ4ZksxMExLNzNGcEM1aE93azJxTVBvQVBNTW9HSVNkSjVTem1WblZYMTNEVVI3NGN1dktsYnhDClZDVmE2OE1LU0pFZlZSRTY0SGE5TFdDVG1TaXJyemFKMUYvdGdwbkc2NnFSdDhMalA0TEY4STgwWGNXVEFEU3cKMHJRdGp0bFFEKy9aVVFLNEptWEQ3V1BjVnRUZWhON0x5WjcvR0VvVVRLR0lQMFZScldndm5iOHNIWEEwQUF0aApFS2ROVk9jUVdTVTlsM000MVFhM1R1eTRCSGhoTVZNUmZ1Y2MzWVZIMFdFVzBacU1JNFgxSEhjNGtjSEdWVXpDCllJRU1OcTdFZlpZeksyMmVsSUkxU2trR3o1WDRrUGtqZkdydFpHV1IKLS0tLS1FTkQgQ0VSVElGSUNBVEUtLS0tLQo=
```

Comment:
The image changed on master, so this test here has to be updated. We'll need a follow-up to fix the issue on master I think.